docs/mast/mast.rst

```diff
@@ -134,6 +134,20 @@ RA and Dec must be given in decimal degrees, and datetimes in MJD.
               image           2          GALEX ... 1000055044                0.0
               image           2          GALEX ... 1000055047 229.81061601101433
 
+We encourage the use of wildcards particularly when querying for JWST instruments
+with the instrument_name criteria. This is because of the varying instrument names
+for JWST science instruments, which you can read more about
+`here <https://outerspace.stsci.edu/display/MASTDOCS/JWST+Instrument+Names>`__.
```

Comment:
I suggest using more descriptive words besides 'here'. For example, "... which you can read more about on the MAST Docs page for `JWST instrument names <link>`__.